pennylane/qnodes/jacobian.py

```diff
@@ -248,7 +248,7 @@ def jacobian(self, args, kwargs=None, *, wrt=None, method="best", options=None):
 
         def inds_using(m):
             """Intersection of ``wrt`` with free params indices whose best grad method is m."""
-            return method_map.get(m, set()).intersection(wrt)
+            return method_map.get(m, set()).intersect(wrt)
```

Comment:
What's the difference here?

Reply:
Shoot, that was an innocent-looking refactor. Will fix.